x-pack/plugins/alerting/server/alerts_client.ts

```diff
@@ -151,7 +139,16 @@ export class AlertsClient {
     this.alertTypeRegistry = alertTypeRegistry;
     this.savedObjectsClient = savedObjectsClient;
     this.createAPIKey = createAPIKey;
-    this.invalidateAPIKey = invalidateAPIKey;
+    this.invalidateAPIKey = async ({ apiKey }: { apiKey: string | null }) => {
+      try {
+        const response = await invalidateAPIKey({ apiKey });
+        if (response.apiKeysEnabled === true && response.result.error_count > 0) {
+          this.logger.error(`Failed to invalidate API Key "${apiKey}"`);
```

Comment:
Are there some error messages we should print here?  Maybe as debug log messages?